config/config.go

```diff
@@ -70,6 +72,8 @@ func Init() error {
 	viper.BindEnv("api-password")
 	viper.BindEnv("service-annotations")
```

Comment:
I'm unsure but do you not need to bind the envs `tls-certificate` and `tls-key`?

```suggestion
	viper.BindEnv("service-annotations")
	viper.BindEnv("tls-certificate")
	viper.BindEnv("tls-key")
```